### PR TITLE
PPML refactor

### PR DESCRIPTION
**Pull request by Litchilitchy**
Some further refactor of PPML

### CODE REVIEW

File: scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/nn/NNAggregator.scala [hunk 1]
```diff
@@ -25,10 +25,10 @@ import com.intel.analytics.bigdl.ppml.common.{Aggregator, FLDataType, FLPhase, S
 import com.intel.analytics.bigdl.ppml.generated.FlBaseProto.Table
 
 abstract class NNAggregator extends Aggregator {
-  protected var hasReturn = false
+  protected var shouldReturn = false
 
-  def setHasReturn(hasReturn: Boolean): Unit = {
-    this.hasReturn = hasReturn
+  def setHasReturn(shouldReturn: Boolean): Unit = {
```
Comment (jason-dai) [inline scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/nn/NNAggregator.scala:30]: `setReturn` or `setShouldReturn`

Comment (Litchilitchy) [reply scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/nn/NNAggregator.scala]: Solved, a mistake..

**Comment by jason-dai** [comment]
LGTM

**Comment by Litchilitchy** [comment]
Table is renamed to TensorMap

**Pull request merged**